sklearn/tests/test_isotonic.py

```diff
@@ -346,3 +346,35 @@ def test_isotonic_zero_weight_loop():
 
     # This will hang in failure case.
     regression.fit(x, y, sample_weight=w)
+
+
+def test_fast_predict():
+    # test that the faster prediction (https://github.com/scikit-learn/scikit-learn/pull/6206)
+    # change doesn't affect out-of-sample predictions.
+    rng = np.random.RandomState(123)
+    N = 10**3
+    # X values over the -10,10 range
+    training_X = 20.0 * rng.rand(N) - 10
+    training_Y = np.less(
+        rng.rand(N),
+        1.0 / (1.0 + np.exp(-training_X))
+        ).astype('int64')
+    
+    slow_model = IsotonicRegression(y_min=0, y_max=1, out_of_bounds="clip")
+    fast_model = IsotonicRegression(y_min=0, y_max=1, out_of_bounds="clip")
+    
```

Comment:
Please avoid introducing new trailing whitespaces (you can use pep8 or flake8 to detect code style violations but also please only fix violations around the lines changed by your PR to keep the diff readable).